src/patternfly/components/DataToolbar/examples/DataToolbar.md

```diff
@@ -309,13 +309,15 @@ The `.pf-m-toggle-group` controls when, and at which breakpoint, filters will be
             Status
           {{/select}}
         {{/data-toolbar-item}}
-        {{> data-toolbar-item-chip-group chip-group--label="Status" chip-group--id=(concat data-toolbar--id '-group1-')}}
         {{#> data-toolbar-item}}
           {{#> select id=(concat data-toolbar--id '-select-checkbox-risk-expanded') select--IsCheckboxSelect="true" select--ItemIsSelected="true"}}
             Risk
           {{/select}}
         {{/data-toolbar-item}}
-        {{> data-toolbar-item-chip-group chip-group--label="Risk" chip-group--id=(concat data-toolbar--id '-group2-')}}
+        {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-chip-container"}}
+          {{> data-toolbar-item-chip-group chip-group--label="Status" chip-group--id=(concat data-toolbar--id '-group1-')}}
```

Comment:
do you need the trailing `-`?

```suggestion
          {{> data-toolbar-item-chip-group chip-group--label="Status" chip-group--id=(concat data-toolbar--id '-group1')}}
```

Reply:
the trailing dash constructs a complex `id` correctly